File: padelManager-backend/src/main/java/com/saul/padelManager/gestionReservas/model/PistaDetallada.java
package com.saul.padelManager.gestionReservas.model;

import com.saul.padelManager.gestionUsuarios.model.Usuario;

import java.util.List;

public record PistaDetallada(int numero, boolean disponible, int huecos, boolean abierta, Long reserva_id, List<Usuario> usuarios) {}

File: padelManager-frontend/src/app/reservas/form-reservar/formulario-reservas/PistaDetallada.ts
import { Usuario } from "src/app/usuarios/usuario";

export interface PistaDetallada {
  numero: number;
  disponible: boolean;
  huecos: number;
  abierta: boolean;
  reserva_id: number | null;
  usuarios: Usuario[];
  pista_propia: boolean;
}